utils/lit/lit.py

```diff
@@ -1,7 +1,6 @@
-#!/usr/bin/env python
```

Comment:
Permission were accidentally changed `100644 → 100755` for `utils/lit/lit.py` and `utils/lit/lit/main.py`. Need to get reverted.